chat-plugins/poll.js

```diff
@@ -33,6 +36,8 @@ class Poll {
 		let ip = user.latestIp;
 		let userid = user.userid;
 
+		if (!user.autoconfirmed) return user.sendTo(this.room, "You need to be autoconfirmed to vote in polls.");
```

Comment:
Why do we need to limit voting to AC users? Tbh if we did thats more of a policy decision.